### PR TITLE
Aspectj remove duplicate check

### PR DESCRIPTION
**Pull request by 0704681032**
when codes execute here ,we can make sure that the exception is not belongsTo exceptionsToIgnore and is belongsTo exceptionsToTrace!
so,we do not need check the exception again
instead,we should call the traceException method directly

### CODE REVIEW

File: sentinel-extension/sentinel-annotation-aspectj/src/main/java/com/alibaba/csp/sentinel/annotation/aspectj/SentinelResourceAspect.java [hunk 1]
```diff
@@ -64,7 +64,7 @@ public Object invokeResourceWithSentinel(ProceedingJoinPoint pjp) throws Throwab
                 throw ex;
             }
             if (exceptionBelongsTo(ex, annotation.exceptionsToTrace())) {
-                traceException(ex, annotation);
+                traceException(ex);//exceptionsToIgnore not include ex &&  exceptionsToTrace include ex
```
Comment (sczyh30) [inline sentinel-extension/sentinel-annotation-aspectj/src/main/java/com/alibaba/csp/sentinel/annotation/aspectj/SentinelResourceAspect.java]: Maybe the comment is not needed here?

Comment (0704681032) [reply sentinel-extension/sentinel-annotation-aspectj/src/main/java/com/alibaba/csp/sentinel/annotation/aspectj/SentinelResourceAspect.java]: sorry to interrupt you,it is my first pr :( 
i create a new branch,and set git config --global user.email "307517019@qq.com",but it still include the first pr(at that time,i did not set the git config: email),so i cannot pass the cla check

Comment (sczyh30) [reply sentinel-extension/sentinel-annotation-aspectj/src/main/java/com/alibaba/csp/sentinel/annotation/aspectj/SentinelResourceAspect.java]: You may need to squash your commits with your correct email (you could leverage `git rebase` with squash mode).

Comment (sczyh30) [reply sentinel-extension/sentinel-annotation-aspectj/src/main/java/com/alibaba/csp/sentinel/annotation/aspectj/SentinelResourceAspect.java]: And you could just drop the last commit during your `rebase`.

**Comment by xjxia** [comment]
need do a new pr?

**Pull request closed without merging**